README.md

```diff
@@ -226,7 +226,11 @@ Here's what the full _default_ module configuration looks like:
     // Sessions aren't pinned to the user's IP address
     ipPinning: false,
     // Expiration of the sessions are not reset to the original expiryInSeconds on every request
-    rolling: false
+    rolling: false,
+    // Uninitialized, resp. unmodified sessions are saved to the store, so session cookies are set at the first response
+    saveUninitialized: true,
+    // Sessions are saved to the store, even if they were never modified during the request
+    resave: true
```

Comment:
- Same here, you need to provide better explanation
- Also `expressjs/session` recommends false by default

Reply:
`expressjs/session` recommends false, but the current behavior of `@sidebase/nuxt-session` is true, so I've chosen to not change the default behavior. It is also recommended to set `resave` to false.

We can change it if you want.